src/test.py

```diff
@@ -25,12 +25,9 @@
 from docker_helpers import get_image_name, pull
 
 
-def run_in_teamcity_block(name, work):
-    print("##teamcity[blockOpened name='{name}']".format(name=name))
-    try:
-        work()
-    finally:
-        print("##teamcity[blockClosed name='{name}']".format(name=name))
+def run_in_buildkite_block(name, work):
+    print("--- " + name)
```

Comment:
I didn't know you could do this to get buildkite to recognise different blocks! nice